File: Solution-iOS/MuslimLib/Source/Models/Trie/ContentFileValueFactory.m
//
//  ContentFileValueFactory.m
//  TipBasedApp
//
//  Created by Mohammad Ali Yektaie on 8/17/15.
//  Copyright (c) 2015 Mohammad Ali Yektaie. All rights reserved.
//

#import "ContentFileValueFactory.h"

@implementation ContentFileValueFactory

- (id<ITrieIndexValue>)createNew {
    return [[ContentFile alloc] init];
}

@end

File: ContentCreator/src/com/yekisoft/muslim_lib/core/trie/content/ContentFileValueFactory.java
package com.yekisoft.muslim_lib.core.trie.content;

import com.yekisoft.muslim_lib.core.trie.ITrieIndexValue;
import com.yekisoft.muslim_lib.core.trie.IValueFactory;

/**
 * Created by yektaie on 8/5/16.
 */
public class ContentFileValueFactory implements IValueFactory {

    @Override
    public ITrieIndexValue CreateNew() {
        return new ContentFile();
    }

}